pkg/abstract.go

```diff
@@ -28,7 +28,7 @@ func scrapeAwsData(config ScrapeConf, now time.Time, metricsPerQuery int, fips,
 					result, err := clientSts.GetCallerIdentity(&sts.GetCallerIdentityInput{})
 					if err != nil {
 						log.Printf("Couldn't get account Id for role %s: %s\n", role.RoleArn, err.Error())
-
+						return
```

Comment:
Just checking, is this added on purpose?

Reply:
Good catch, this was some other pending code I forgot to back out.